src/Microsoft.Health.Dicom.Core.UnitTests/Features/CustomTag/GetCustomTagsServiceTests.cs

```diff
@@ -42,8 +42,8 @@ public async Task GivenRequestForAllTags_WhenNoTagsAreStored_ThenExceptionShould
         [Fact]
         public async Task GivenRequestForAllTags_WhenMultipleTagsAreStored_ThenCustomTagEntryListShouldBeReturned()
         {
-            CustomTagStoreEntry tag1 = CreateCustomTagEntry(1, "45456767", DicomVRCode.AE.ToString(), CustomTagLevel.Instance, CustomTagStatus.Added);
-            CustomTagStoreEntry tag2 = CreateCustomTagEntry(2, "01012323", DicomVRCode.FL.ToString(), CustomTagLevel.Series, CustomTagStatus.Reindexing);
+            CustomTagStoreEntry tag1 = CreateCustomTagEntry(1, "45456767", DicomVRCode.AE.ToString(), null, CustomTagLevel.Instance, CustomTagStatus.Added);
```

Comment:
We should probably have an example where the private tag creator is supplied

Reply:
* This task focus on adding new column in database, coming task will add more unit test for coverage
* This unit test verifies GetCustomTagService API returns correct data, and without coming task (https://microsofthealth.visualstudio.com/Health/_boards/board/t/Medical%20Imaging/Stories/?workitem=80425), we wont' be able to do it.